lib/main.dart

```diff
@@ -1,14 +1,14 @@
 import 'package:flutter/material.dart';
-import 'package:sentry_flutter/sentry_flutter.dart';
+import 'package:sentry/sentry.dart';
 
 import 'HomePage.dart';
 
 Future<void> main() async {
-  await SentryFlutter.init(
```

Comment:
@mskarkala It looks like this is a different package, that doesn't have Flutter support; only Dart. See here: https://pub.dev/packages/sentry#flutter

I'll try to use `sentry_flutter` some more